docs/workflow.md

```diff
@@ -0,0 +1,137 @@
+
+# Workflows
+
+Ths document documents two different yet common workflows in BKR using `kubeprod` and `kubecfg`:
+
+* Basic workflow
```

Comment:
Are we going to point to the quickstarts for the basic workflow?

Reply:
Sounds like a plan.